### ISSUE TITLE
Trend Micro slows Atom startup considerably after Atom update

### ISSUE THREAD

**Issue by rlee287** [atom/atom#12601]
### Prerequisites

* [x] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode)?
* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [x] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported? **More generic issue reported at #2654**

For more information on how to write a good [bug report](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report) or [enhancement request](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion), see the `CONTRIBUTING` guide.

### Description

Atom starts slowly when Trend Micro Maximum Security Antivirus is enabled. This happened only after an upgrade fom 1.9.0 to 1.10.0. Even with the atom folder added to the exception list, the problem still occurs.

### Steps to Reproduce

1. Start up atom with the following script:

```python
from time import perf_counter# Python timer function
import os

begin=perf_counter()# Start timer
os.system("atom --safe --profile-startup")# Exits when cmd prompt would usually return
end=perf_counter()# End timer
print("Time",end-begin)# Print time delta
```

2. Save the startup profile
3. Open a `timecop` view and screenshot times
4. Copy the printed time

These are logs collected with `--profile-startup` (renamed to have .txt extension):

[CPU-20160903T140657_antivirus_safe.cpuprofile.txt](https://github.com/atom/atom/files/453636/CPU-20160903T140657_antivirus_safe.cpuprofile.txt)

[CPU-20160903T140306_noantivirus_safe.cpuprofile.txt](https://github.com/atom/atom/files/453635/CPU-20160903T140306_noantivirus_safe.cpuprofile.txt)

These are the screenshots with `timecop`:

**Antivirus:**
![atom_antivirus](https://cloud.githubusercontent.com/assets/14067959/18227639/7ef48a18-71e0-11e6-99d6-251c5b0a4602.PNG)

**No Antivirus:**
![atom_no_antivirus](https://cloud.githubusercontent.com/assets/14067959/18227645/8ccac36e-71e0-11e6-8662-6436afae5324.PNG)

These are the times returned by the python script (precise to at least a microsecond):

**Antivirus:** 11.949334730807175
**No Antivirus:** 0.2880780427960264

The Atom process didn't even appear in Task Manager for over 10 seconds (if this helps to pin down the cause).

**Expected behavior:** Fast startup

**Actual behavior:** Startup that takes over 10 seconds

### Versions
Windows 10
```
> atom --version

Atom    : 1.10.0
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0
```
```
> apm --version
apm  1.12.5
npm  3.10.5
node 4.4.5
python 3.5.2
git 2.5.3.windows.1
visual studio 2013
```
Perhaps the discrepancy in the `node` versions may have something to do with this as well?

**Comment by Zireael07** [comment]
I believe we had an issue to do with the game window not appearing for a couple of seconds?

**Comment by rlee287** [comment]
I am aware of the more generic issue of #2654. However, this conflict with Trend Micro is more specific, so I decided to file this as a separate issue.

**Comment by damieng** [comment]
Which Atom folder did you add to the exclusion list?  You could try adding Atom in %localappdata% and .Atom in %userprofile%

**Comment by rlee287** [comment]
I did that before already, adding `C:\Users\<USERNAME>\AppData\Local\atom` as a folder, but it didn't seem to affect anything. However, when I opened atom just now, it worked properly (without me having to do anything). It seems that some update to Trend Micro fixed a bug in the exception list functionality. Since this turned out to be a problem on Trend Micro's end, I will close this issue now.
